pkg/health/clusterStatsHandler.go

```diff
@@ -127,34 +130,40 @@ func calculateStorageSavedPercentage(incomingBytes, onDiskBytes float64) float64
 	return storageSaved
 }
 
-func convertDataToSlice(indexToAllIndexStats map[string]utils.AllIndexesStats, volumeField, countField,
+func convertDataToSlice(allIndexStats utils.AllIndexesStats, volumeField, countField,
 	segmentCountField string) []map[string]map[string]interface{} {
 
+	indices := make([]string, 0)
+	for index := range allIndexStats.IndexToStats {
+		indices = append(indices, index)
+	}
+	sort.Strings(indices)
```

Comment:
do we need to sort ?

Reply:
it would make the results look more consistent.